projects/ampproject/Dockerfile

```diff
@@ -14,7 +14,7 @@
 #
 ################################################################################
 
-FROM gcr.io/oss-fuzz-base/base-builder
+FROM gcr.io/oss-fuzz-base/base-builder@sha256:19782f7fe8092843368894dbc471ce9b30dd6a2813946071a36e8b05f5b1e27e
```

Comment:
It would be really incredible if you could leave a comment on each of these pinnings where we link back to this PR, and in this PR you could explain how to find the build failure logs.